packages/terra-clinical-data-grid/src/subcomponents/Cell.jsx

```diff
@@ -133,7 +135,7 @@ class Cell extends React.Component {
         aria-selected={isSelected ? true : undefined}
       >
         <div
-          role="button"
+          role={isSelectable ? 'button' : undefined}
```

Comment:
if we are adding this role based on child content, should we be validating the children count?
```
React.Children.count(children) > 0

Reply:
Let's let the implementer disable/enable the cells like they have been and just make sure our internal attributes match up. This implementation should be fine, but the CHANGELOG needs to be updated.

Reply:
Updated the CHANGELOG entry here https://github.com/cerner/terra-clinical/pull/706/commits/77a864a57d2fbb2fdc523817326c1b81f69d216a